packages/actor-rdf-metadata-extract-void/README.md

```diff
@@ -20,7 +20,7 @@ $ yarn add @comunica/actor-rdf-metadata-extract-void
 
 This actor adds the following entries to the metadata object.
 
-* `datasets`: An array containing the extracted VoID dataset metadata.
+* `patternCardinalityEstimator`: Function to provide cardinality estimates for triple patterns.
```

Comment:
Why was this changed?
This `datasets` object looked like a nice design to me.
`patternCardinalityEstimator` seems like it can lead to more unintended conflicts, as it won't allow different actors to produce information for different datasets.